client/me/help/help-contact/index.jsx

```diff
@@ -286,7 +295,11 @@ module.exports = React.createClass( {
 		//Autofill the subject field since we will be showing it now that operators have went away.
 		this.autofillSubject();
 
-		this.showAvailabilityNotice( IS_UNAVAILABLE );
+		if ( timedOut ) {
```

Comment:
@gwwar you have to be signed into Olark as an operator, and then you have to force the API to send back the group id for a group your operator belongs to. Then you can toggle status. If you'd like help testing this, just ping me and I can log into chat and help you verify.